espnet2/train/trainer.py

```diff
@@ -820,6 +820,10 @@ def plot_attention(
 
                     if att_w.ndim == 2:
                         att_w = att_w[None]
+                    if att_w.ndim == 4:
+                        att_w = np.concatenate(
+                            [att_w[i] for i in range(att_w.shape[0])], axis=0
+                        )
                     elif att_w.ndim > 3 or att_w.ndim == 1:
                         raise RuntimeError(f"Must be 2 or 3 dimension: {att_w.ndim}")
```

Comment:
Better to update the error message as well.

Reply:
cool.